debian/control

```diff
@@ -7,7 +7,7 @@ Build-Depends: gcc (>=5), ninja-build, meson (>=0.40),
  libgstreamer1.0-dev, libgstreamer-plugins-base1.0-dev, libglib2.0-dev,
  gstreamer1.0-tools, gstreamer1.0-plugins-base, gstreamer1.0-plugins-good,
  libgtest-dev, ssat, libpng-dev, libopencv-dev, liborc-0.4-dev,
- python, python-numpy,
+ python, python-numpy, python3, python3-dev, python3-numpy,
```

Comment:
Write the commit title in order that reviewers can understand a goal of the PR at one.
* before: [PythonFilter/Build] Update build/packaging files
* after   : [Build] PythonFilter: Update build files for Ubuntu packaging and Meson build

Reply:
Got it.